android-project/app/src/main/java/oceantreasur/es/EndGameActivity.java

```diff
@@ -40,30 +55,82 @@ public void onClick(View v) {
         });
 
         if (ViewUtils.isTablet(this)) {
+            RelativeLayout mainlayout = (RelativeLayout) findViewById(R.id.rl_end);
             ImageView logo = (ImageView) findViewById(R.id.iv_logo_end);
-            final ImageView fish1 = (ImageView) findViewById(R.id.fish1);
-            final ImageView fish2 = (ImageView) findViewById(R.id.fish2);
-            final ImageView chest = (ImageView) findViewById(R.id.chest_end);
-
-            View.OnClickListener animate = new View.OnClickListener() {
-                @Override
-                public void onClick(View v) {
-                    moveAnimation(fish1, fish2);
-                }
-            };
-
-            logo.setOnClickListener(animate);
-            chest.setOnClickListener(animate);
+            ImageView chest = (ImageView) findViewById(R.id.chest_end);
+
+            LayoutInflater Li = LayoutInflater.from(getApplicationContext());
+
+            for(int i = 0; i < MAX_FISH; i++) {
+                fishArr.add(Li.inflate(fish,null));
+                mainlayout.addView(fishArr.get(i));
+            }
+
+            moveAnimation();
         }
     }
 
-    private void moveAnimation(ImageView fish1, ImageView fish2) {
-        Animation animation = new TranslateAnimation(Animation.ABSOLUTE, 300, Animation.ABSOLUTE, Animation.ABSOLUTE);
-        animation.setDuration(2500);
-        animation.setFillAfter(true);
+    public static int getScreenWidth() {
+        return Resources.getSystem().getDisplayMetrics().widthPixels;
+    }
+
+    public static int getScreenHeight() {
+        return Resources.getSystem().getDisplayMetrics().heightPixels;
+    }
+
+    private int choseRandomYPosition() {
+        int height = getScreenHeight();
+        Random rand = new Random();
+
+        int yPosition = rand.nextInt(height) + 1;
+
+        if(yPosition < height * (5/100)) {
```

Comment:
Turn 5/100 and 95/100 into constants. You don't need to calculate them every iteration.